### PR TITLE
[ModelZoo] Add DCN Model.

### PR DESCRIPTION
**Pull request by LALBJ**

### CODE REVIEW

**Review by shanshanpt — COMMENTED** [review]
Should refine commit title.

File: modelzoo/DCN/train.py [hunk 1]
```diff
@@ -0,0 +1,916 @@
...
+    'C26': 128
+}
+
+
+class DCN():
+    def __init__(self,
+                 emb_stacking_columns=None,
+                 dnn_hidden_units=[1024, 512, 256],
+                 optimizer_type='adam',
+                #  linear_learning_rate=0.2,
```
Comment (shanshanpt) [inline modelzoo/DCN/train.py:102]: delete useless code

File: modelzoo/DCN/train.py [hunk 2]
```diff
@@ -0,0 +1,916 @@
...
+    parser.add_argument('--seed',
+                        help='set the random seed for tensorflow',
+                        type=int,
+                        default=2021)
+    parser.add_argument('--optimizer',
+                        type=str, \
+                        choices=['adam', 'adamasync', 'adagraddecay', 'adagrad'],
+                        default='adamasync')
+    # parser.add_argument('--linear_learning_rate',
+    #                     help='Learning rate for linear model',
```
Comment (shanshanpt) [inline modelzoo/DCN/train.py:736]: delete useless code

**Review by shanshanpt — COMMENTED** [review]
现在modelzoo中模型目录名称都改成小写了 ，这里需要修改一下：
modelzoo/dcn

**Review by shanshanpt — COMMENTED** [review]
Delete the useless directory 'DCN', it's instead by 'dcn'.

**Comment by LALBJ** [comment]
> 现在modelzoo中模型目录名称都改成小写了 ，这里需要修改一下： modelzoo/dcn

done

**Review by shanshanpt — APPROVED** [review]
LGTM

**Pull request merged**